git-send-email.perl

```diff
@@ -601,14 +615,14 @@ sub is_format_patch_arg {
 	if ($f eq "--") {
 		push @rev_list_opts, "--", @ARGV;
 		@ARGV = ();
-	} elsif (-d $f and !is_format_patch_arg($f)) {
+	} elsif (-d $f and (!is_format_patch_arg($f))) {
```

Comment:
Well, if you've ever used -pedantic in gcc, you know that it encourages parens even when the grouping is seemingly obvious.  Perlcritic is the same way... It does occasionally generate some 'noise'... If the tool is to be useful, you find a way to get rid of the noise so that all that is left is meaningful messages.  That might mean throwing in extraneous parens to shut it up.